torchprime/launcher/buildpush.py

```diff
@@ -71,8 +71,8 @@ def buildpush(
     # Use the provided base image
     base_image = base_docker_url
   else:
-    # Use torch_xla Python 3.10 as the base image
-    base_image = f"us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_3.10_tpuvm_{torch_xla_version}"
+    # Use torch_xla Python 3.12 as the base image
+    base_image = f"us-central1-docker.pkg.dev/tpu-pytorch-releases/docker/xla:nightly_3.12_tpuvm_{torch_xla_version}"
```

Comment:
The torch_xla_version comes from the file https://github.com/AI-Hypercomputer/torchprime/blob/main/pyproject.toml#L51C22-L51C30

See line 67 in this file.

Can we update that to `20250715`?  I don't see 3.12 nightly torch_xla docker build for the date listed in that file (`20250617`)